website/docs/user-guide/_category_.yml

```diff
@@ -0,0 +1,2 @@
+position: 3
```

Comment:
Please run `pnpm fornat` in the website directory for this commit to fix any formatting issues.